apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -222,6 +230,40 @@ public override int ExecuteNonQuery() => ExecuteNonQueryAsync(_synchronousCancel
                     "You can only call ExecuteNonQueryAsync on a Delete, Insert, InsertUpdate, or Update Command");
             }
 
+            if (SpannerCommandTextBuilder.SpannerCommandType == SpannerCommandType.Ddl)
+            {
+                return ExecuteDdlAsync(cancellationToken);
+            }
+            else
+            {
+                return ExecuteMutationsAsync(cancellationToken);
+            }
+        }
+
+        private async Task<int> ExecuteDdlAsync(CancellationToken cancellationToken)
+        {
+            var databaseAdminClient = await DatabaseAdminClient.CreateAsync();
```

Comment:
I'll do it right after checking in these two.  Then I just have my in progress doccomments which (you would hope) wont have merge conflicts.